dist/jquery.panzoom.js

```diff
@@ -1196,7 +1202,8 @@
 
 			var move = function(e) {
 				var coords;
-				e.preventDefault();
+				//e.preventDefault();  // chrome passive event warning https://www.chromestatus.com/features/5093566007214080  #328
+				e.stopPropagation();
```

Comment:
You can add a comment that the use of stopPropagation is because preventDefault is not allowed.. But don't comment out the old code.